log/internal/logger/logger.go

```diff
@@ -0,0 +1,96 @@
+package logger
```

Comment:
this file is similar to https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/lib/log/internal/globallogger/globallogger.go